TSPL.docc/LanguageGuide/Macros.md

```diff
@@ -218,7 +218,7 @@ including the attributes for its roles:
 
 ```swift
 @attached(member)
-@attached(conformance)
+@attached(extension)
```

Comment:
This should specify the conformances that are added, e.g.

```swift
@attached(extension, conformances: OptionSet)
```

Reply:
Fixed in d444c690.